docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1748 @@
+              [ 0.5+0.j, -0.5+0.j,  0.5+0.j, -0.5+0.j],
+              [ 0.5+0.j,  0.5+0.j, -0.5+0.j, -0.5+0.j],
+              [ 0.5+0.j, -0.5+0.j, -0.5+0.j,  0.5+0.j]],
+             input_dims=(2, 2), output_dims=(2, 2))
+
+    # or...
+    >>> qc = QuantumCircuit(1)
+    >>> qc.h(0)
+    >>> H = Clifford(qc).to_operator()
+    >>> operator = H ^ H
+    >>> operator
+    Operator([[ 0.5+0.j,  0.5+0.j,  0.5+0.j,  0.5+0.j],
+              [ 0.5+0.j, -0.5+0.j,  0.5+0.j, -0.5+0.j],
+              [ 0.5+0.j,  0.5+0.j, -0.5+0.j, -0.5+0.j],
+              [ 0.5+0.j, -0.5+0.j, -0.5+0.j,  0.5-0.j]],
+             input_dims=(2, 2), output_dims=(2, 2))
+
+
+.. raw:: html
+
+   </details>
+
+1-Qubit States
+--------------
+*Back to* `Contents`_
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+
+   * - :class:`~qiskit.opflow.Zero`, :class:`~qiskit.opflow.One`, :class:`~qiskit.opflow.Plus`, :class:`~qiskit.opflow.Minus`
+     - :class:`~qiskit.quantum_info.Statevector` or simply :class:`~qiskit.circuit.QuantumCircuit`, depending on the use case.
+
+       ..  note::
+
+           For efficient simulation of stabilizer states, :mod:`~qiskit.quantum_info` includes a
+           :class:`~qiskit.quantum_info.StabilizerState` class. See API ref. for more info.
+
+.. raw:: html
+
+    <details>
+    <summary><a><font size="+1">Example 1: Working with stabilizer states</font></a></summary>
+    <br>
+
+**Opflow**
+
+.. code-block:: python
+
+    >>> from qiskit.opflow import Zero, One, Plus, Minus
+
+    # Zero, One, Plus, Minus are all stabilizer states
+    >>> state1 = Zero ^ One
+    >>> state2 = Plus ^ Minus
+
+    >>> state1
+    DictStateFn({'01': 1}, coeff=1.0, is_measurement=False)
+
+    >>> print(state2)
+    CircuitStateFn(
+         ┌───┐┌───┐
+    q_0: ┤ X ├┤ H ├
+         ├───┤└───┘
+    q_1: ┤ H ├─────
+         └───┘
+    )
+
+**Alternative**
+
+.. code-block:: python
+
+    >>> from qiskit import QuantumCircuit
+    >>> from qiskit.quantum_info import StabilizerState, Statevector
+
+    >>> qc_zero = QuantumCircuit(1)
+    >>> qc_one = qc_zero.copy()
+    >>> qc_one.x(0)
+    >>> state1 = Statevector(qc_zero) ^ Statevector(qc_one)
+
+    >>> qc_plus = qc_zero.copy()
+    >>> qc_plus.h(0)
+    >>> qc_minus = qc_one.copy()
+    >>> qc_minus.h(0)
+    >>> state2 = StabilizerState(qc_plus) ^ StabilizerState(qc_minus)
+
+    >>> state1
+    Statevector([0.+0.j, 1.+0.j, 0.+0.j, 0.+0.j],
+                dims=(2, 2))
+
+    >>> state2
+    StabilizerState(StabilizerTable: ['-IX', '+XI'])
+
+.. raw:: html
+
+   </details>
+
+
+Primitive and List Ops
+======================
+*Back to* `Contents`_
+
+Most of the workflows that previously relied in components from :mod:`~qiskit.opflow.primitive_ops` and
+:mod:`~qiskit.opflow.list_ops` can now leverage elements from :mod:`~qiskit.quantum_info`\'s
+operators instead.
+Some of these classes do not require a 1-1 replacement because they were created to interface with other
```

Comment:
I would lean somewhat towards removing this last sentence as each of the classes are covered below, and since its rather vague as to what the classes in question might be. I think you cover all the classes in this category that comprise the public API - if its internal to opflow then at this level it should not be a consideration right.